assignments/subscriptions/2307d175-7e49-434b-9ac2-515529b845f2/assign.appgateway_diagnostics.json

```diff
@@ -0,0 +1,30 @@
+{
+    "properties": {
+        "metadata": {
+            "assignedBy": "PlatOps"
```

Comment:
we've started using the repo name (should probably update all of them)

https://github.com/hmcts/azure-policy/blob/92d270357c4e40a54cd4eb9268b93d667db6a11d/assignments/mgmt-groups/mg-dts002/assign.keyvault.json#L4